lib/usd/hdMaya/adapters/adapter.h

```diff
@@ -16,16 +16,16 @@
 #ifndef HDMAYA_ADAPTER_H
 #define HDMAYA_ADAPTER_H
 
-#include <hdMaya/api.h>
-#include <hdMaya/delegates/delegateCtx.h>
-
 #include <vector>
 
 #include <maya/MMessage.h>
 
 #include <pxr/pxr.h>
 #include <pxr/usd/sdf/path.h>
 
+#include "hdMaya/api.h"
+#include "hdMaya/delegates/delegateCtx.h"
+
```

Comment:
`hdMaya/api.h` and `hdMaya/delegates/delegateCtx.h` are both installed, and so are public, and should be formatted how they were previously (only the order needed to be changed):

```
#include <hdMaya/api.h>	
#include <hdMaya/delegates/delegateCtx.h>
```